contracts/src/v0.8/functions/dev/v1_X/FunctionsBilling.sol

```diff
@@ -268,9 +306,24 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
       response,
       err,
       juelsPerGas,
-      gasOverheadJuels + commitment.donFee, // cost without callback or admin fee, those will be added by the Router
+      // The following line represents: "cost without callback or admin fee, those will be added by the Router"
+      // But because the _offchain_ Commitment is using operation fee in the place of the admin fee, this now adds admin fee (actually operation fee)
+      // Admin fee is configured to 0 in the Router
+      gasOverheadJuels + commitment.donFee + commitment.adminFee,
```

Comment:
Alternatively we could have a variable named operationFee above (set to adminFee). Then this line will be clearer.